pkg/catalog/catalog.go

```diff
@@ -56,9 +57,23 @@ func Load(dir string, validChartRefs []string) (*Catalog, error) {
 		return nil, fmt.Errorf("matching files with glob expression %s: %w", globExpr, err)
 	}
 
+	// Load .joyignore if it exists
+	ignoreFilePath := filepath.Join(dir, ".joyignore")
```

Comment:
You do not need to stat here. It would read much nicer if:
```go
joyIgnore, err := ignore.CompileIgnoreFile(filepath.Join(dir, ".joyignore")
if err != nil && !errors.Is(err, os.ErrNotExists) {
  return nil, fmt.Errror(...)
}
```